.github/workflows/ci.yml

```
@@ -126,7 +90,7 @@ jobs:
       
 
   test-warp-cpu:
-    if: github.event_name != 'schedule' || github.repository == 'NVIDIA/warp'
+    if: false
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)